src/components/organisms/FundedBy.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styles from './FundedBy.module.css'
+import Container from '../atoms/Container'
+
+const query = graphql`
+  {
+    fundedBy: allFile(
+      filter: { absolutePath: { regex: "/src/images/fundedBy/" } }
+    ) {
+      edges {
+        node {
+          childImageSharp {
+            id
+            original {
+              src
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+interface Logos {
+  fundedBy: {
+    edges: {
+      node: {
+        childImageSharp: {
+          id: string
+          original: {
+            src: string
```

Comment:
suggestion: for responsive image optimization we could / should probably make use of [gatsby-image](https://www.gatsbyjs.com/plugins/gatsby-image/#two-types-of-responsive-images)